skipper.yaml

```diff
@@ -0,0 +1,16 @@
+registry: quay.io
```

Comment:
Does this matter (is it needed). Just `quay.io` looks a bit random 🤔 Could it be just empty string, or `local` or `registry.k8s.io` WDYT?

Reply:
the goal of "registry" is to define the registry where the skipper-built image can be pushed. But in practice, nobody ever pushes it, people just build the image once, locally, if needed, and then use it. I will change it to local